app/models/event.rb

```diff
@@ -0,0 +1,7 @@
+class Event < ApplicationRecord
+  validates :title , presence:true
```

Comment:
Consider adding a blank line after the class declaration for better readability.